google/cloud/bigtable/rpc_retry_policy.h

```diff
@@ -24,6 +25,25 @@ namespace google {
 namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
+/// An adapter to use `grpc::Status` with the `google::cloud::*Policies`.
+struct SafeGrpcRetry {
+  // Sometimes we need to use google::protobuf::rpc::Status, in which case this
+  // is a easier function
+  static inline bool IsTransientFailure(grpc::StatusCode code) {
+    return code == grpc::StatusCode::OK or code == grpc::StatusCode::ABORTED or
```

Comment:
Seems a bit strange to treat status `OK` as a "transient failure"...

Reply:
Yes.  This was defensive, in case somebody (me most likely) called this with an `StatusCode::OK`.  I created #741 for it.